### ISSUE TITLE
Compilation fails on Linux kernels >= 4.12

### ISSUE THREAD

**Issue by marcelhuberfoo** [pkoutoupis/rapiddisk#19]
Due to kernel changes/cleanup, compilation of the rapiddisk kernel module fails with the following output:

```
/rapiddisk-dkms/usr/src/rapiddisk-5.0 $ make -C $(pwd) M=$PWD
make: Entering directory '/rapiddisk-dkms/usr/src/rapiddisk-5.0'
make -C/lib/modules/4.12.3-1-ARCH/build M=/rapiddisk-dkms/usr/src/rapiddisk-5.0
make[1]: Entering directory '/usr/lib/modules/4.12.3-1-ARCH/build'
  LD      /rapiddisk-dkms/usr/src/rapiddisk-5.0/built-in.o
  CC [M]  /rapiddisk-dkms/usr/src/rapiddisk-5.0/rapiddisk.o
/rapiddisk-dkms/usr/src/rapiddisk-5.0/rapiddisk.c: In function ‘attach_device’:
/rapiddisk-dkms/usr/src/rapiddisk-5.0/rapiddisk.c:708:26: error: ‘struct queue_limits’ has no member named ‘discard_zeroes_data’
  rdsk->rdsk_queue->limits.discard_zeroes_data = 1;
                          ^
make[2]: *** [scripts/Makefile.build:309: /rapiddisk-dkms/usr/src/rapiddisk-5.0/rapiddisk.o] Error 1
make[1]: *** [Makefile:1512: _module_/rapiddisk-dkms/usr/src/rapiddisk-5.0] Error 2
make[1]: Leaving directory '/usr/lib/modules/4.12.3-1-ARCH/build'
make: *** [Makefile:32: all] Error 2
make: Leaving directory '/rapiddisk-dkms/usr/src/rapiddisk-5.0'
```
According to this [mailing list entry](https://www.spinics.net/lists/raid/msg57009.html) the flag can completely be removed.

**Comment by pkoutoupis** [comment]
@marcelhuberfoo, 
Please see my notes on your pull request.

**Comment by pkoutoupis** [comment]
Addressed with commit: 354c7596f1cf85b3e3240129e8d9d126ec25c522
